README.md

```diff
@@ -314,13 +314,13 @@ environment variable:
   ensure alignment.  (The standard library `align_to` method works fine in both
   modes; under symbolic alignment it only fills the middle slice when the
   allocation guarantees sufficient alignment.)
-* `-Zmiri-track-alloc-id=<id>` shows a backtrace when the given allocation is
+* `-Zmiri-track-alloc-id=<id-list>` shows a backtrace when the given allocations are
```

Comment:
I think `<id1>,<id2>,...` would be clearer about how the list is supposed to be written.